### PR TITLE
📖 Improve getting started guide and tilt dev guide

### LINKED ISSUE #80: 📖 Update getting started guide to give more information of the workflow
/kind doc

Right now the getting started section of the doc is very abstract and making it more informative for the user will help.

### PR DESCRIPTION
**Pull request by aniruddha2000**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #80 

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [x] squash commits
- [x] include documentation

### CODE REVIEW

**Comment by aniruddha2000** [comment]
TODO: There should be some information about tearing down the workload cluster

File: docs/book/src/user/getting-started.md [hunk 1]
```diff
@@ -73,5 +99,5 @@ Please have a look at the [Developer Guide](../developer/index.md), if you want
 
 ## Navigation in the docs
 
-On the left and right side of the documentation you see angle brackets. You can use them to switch 
+On the left and right side of the documentation you see angle brackets. You can use them to switch
 to the next/previous page.
```
Comment (batistein) [inline docs/book/src/user/getting-started.md:103]: Why is this in a new line?

**Review by batistein — DISMISSED** [review]

File: Makefile [hunk 2]
```diff
@@ -458,6 +458,13 @@ create-mgt-cluster: $(CTLPTL) ## Creates kind-dev Management Cluster
 	rm -f $(CAPHV_WORKER_CLUSTER_KUBECONFIG) $(CAPHV_MGT_CLUSTER_KUBECONFIG)
 	./hack/kind-dev.sh
 
+.PHONY: delete-workload-cluster
+delete-workload-cluster: ## Deletes the example workload Kubernetes cluster
+	@test $${CLUSTER_NAME?Please set environment variable}
+	@echo 'Your Hivelocity resources will now be deleted.'
+	kubectl wait --for=delete cluster/$(CLUSTER_NAME) --timeout=15m
```
Comment (aniruddha2000) [inline Makefile:465]: Just noticed `k delete cluster/hvtest-aniruddha` deletes the cluster successfully but `k wait --for=delete cluster/hvtest-aniruddha --timeout=10m` doesn't

Comment (guettli) [reply Makefile:465]: @aniruddha2000 `k wait ...` does nothing but waiting. You need two lines: the first line deletes the cluster, the second waits until the action of the first line was done. Please update the PR. thank you.

**Pull request merged**